iomgr/lib/dbengine/handlers/RequestHandler.h

```diff
@@ -578,6 +579,25 @@ class RequestHandler final {
      */
     void writeJsonEpilog(siodb::io::JsonWriter& jsonWriter);
 
+public:
```

Comment:
why this should have been made public?

Reply:
@cxxman Because they're static constant so they will never be set.
Should it be private with a getter?

Reply:
No. Now I've found that they are externally used in the class IOManagerRestConnectionHandler. I think then it is better to move them into separate enum. I'll create a followup issue for myself to do it later. For now let's leave this as is.